backend/uclapi/roombookings/decorators.py

```diff
@@ -24,6 +26,66 @@ def wrapped(request, *args, **kwargs):
             return response
 
         try:
+            if token.split("-")[1] == "temp":
+                is_temp_token = True
+        except IndexError:
+            is_temp_token = False
+
+        if is_temp_token:
```

Comment:
this could throw an undefined error, because in one case the `is_temp_token` can go undefined